src/components/buttons/ButtonsContainer.tsx

```diff
@@ -57,7 +57,7 @@ export const ButtonsContainer = React.memo(({ absolute = false, fade, primary, s
     <Animated.View style={fadeStyle}>
       <StyledButtonContainer absolute={absolute} layout={layout}>
         {renderButton('primary', primary)}
-        {renderButton('secondary', secondaryAsPrimary)}
+        {renderButton('secondary', secondaryRight)}
```

Comment:
Technically not always to the "right". If the layout is vertical it will be below so hence my recommendation of using `secondary2`